doc/source/contrib/language_ref/property_ref/requirement_types.rst

```diff
@@ -103,8 +104,8 @@ In the above example *mypackage* must have a revision between 0.0 and 1.0 or
 
 Cache keys:
 
-  * package - name of each installed package
-  * revision - revision of each installed package
+  * snap - name of each installed snap
```

Comment:
the cache key is actually called "package" not "snap" so you can leave this as is.